pygeoapi/provider/mongo.py

```diff
@@ -63,21 +64,40 @@ def __init__(self, provider_def):
         dbclient = MongoClient(self.data)
         self.featuredb = dbclient.get_default_database()
         self.collection = provider_def['collection']
+        self.datetime_field = (provider_def['datetime_field']
+                               if 'datetime_field' in provider_def
+                               else 'datetime')
         self.featuredb[self.collection].create_index([("geometry", GEOSPHERE)])
+        self.fields = self.get_fields()
+
+    def _get_field_type(self, obj):
```

Comment:
Are you able to use https://github.com/geopython/pygeoapi/blob/master/pygeoapi/util.py#L70 in anyway instead?  No preference, just pointing out its existence which may or may be of use.

Reply:
I just tried with it, but tests crashed due to following error:
E           TypeError: argument of type 'int' is not iterable

pygeoapi/util.py:111: TypeError

...so I didn't change it (at least yet: )